osquery/config/parsers/file_paths.cpp

```diff
@@ -120,10 +121,15 @@ Status FilePathsConfigParserPlugin::update(const std::string& source,
                          << "': " << sql.getMessageString();
             } else {
               for (auto& row : sql.rows()) {
-                std::string name = category.name.GetString();
                 std::string path = row["path"];
-                replaceGlobWildcards(path);
-                Config::get().addFile(source, name, path);
+                if (path.empty()) {
```

Comment:
There is nit. Speaking semantically, this loop just need to read rows and fetch all paths from it. That means there is no reason for row to be non-const. Also, `operator[]` for the `std::map` is non-const and it creates default value in case of key absense. Which is not perfect.

You could de it with iterator:
```c++
for (const auto& row : sql.rows()) {
  auto pathIt = row.find("path");
  if (pathIt == row.end()) {
     LOG(ERROR) << ...
  } else {
     auto path = *pathIt;
     ...
  }
```